builtin/providers/openstack/resource_openstack_fw_firewall_v1.go

```diff
@@ -121,7 +162,11 @@ func resourceFWFirewallV1Read(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
 	}
 
-	firewall, err := firewalls.Get(networkingClient, d.Id()).Extract()
+	result := firewalls.Get(networkingClient, d.Id())
+
+	var firewall Firewall
+	err = result.ExtractInto(&firewall)
```

Comment:
Just a heads up that this _might_ change in the future. What you have is the current working solution -- just something to be mindful of down the road.